packages/sol-compiler/src/compiler.ts

```diff
@@ -173,6 +133,43 @@ export class Compiler {
         const promisedOutputs = this._compileContractsAsync(this._getContractNamesToCompile(), false);
         return promisedOutputs;
     }
+    public async watchAsync(): Promise<void> {
+        console.clear(); // tslint:disable-line:no-console
+        utils.logWithTime('Starting compilation in watch mode...');
+        const watcher = chokidar.watch('^$', { ignored: /(^|[\/\\])\../ });
```

Comment:
Please set `'^$'` to a descriptive variable name. It's not immediately clear to the reader what this means. Same goes for the regexp.